src/dataTransfer/dataTransferUploadBuffer.ts

```diff
@@ -17,18 +17,45 @@ const debug = debug0('atem-connection:data-transfer:upload-buffer')
 export interface UploadBufferInfo {
 	encodedData: Buffer
 	rawDataLength: number
+	isRleEncoded: boolean
 	hash: string | null
```

Comment:
Could we add some descriptions to the properties in this interface? Especially the hash and the intention of that is a bit confusing for a user calling `uploadStill()`.